app/views/subtopics/show.html.erb

```diff
@@ -18,6 +19,17 @@
   <%= render "govuk_publishing_components/components/title", title_params %>
 <% end %>
 
+<% if subtopic.recruitment_survey_url %>
+  <div class="govuk-width-container">
```

Comment:
We should be able to update this div element from `<div class="govuk-width-container">` to `<div class="govuk-!-static-margin-top-4">`

Using the spacing class from the design system will allow us to remove the `intervention.scss` component stylesheet and the related line in the `manifest.js` file.

This will also resolve a layout issue caused by the presence of the "govuk-width-container" class, when the screen width is less than 1020px, please see the screenshot below showing additional margin on the left and right:
<img width="987" alt="Screenshot 2023-09-26 at 17 12 23" src="https://github.com/alphagov/collections/assets/28779939/bdfa3fcb-6b18-4df9-8fb0-d2ee9edde4e5">